internal/gen-known-logs/Makefile

```diff
@@ -0,0 +1,5 @@
+.PHONY: ctlogs
+
+ctlogs:
+	go run github.com/square/certigo/internal/gen-known-logs --package lib > ../../lib/ctlogs.go
```

Comment:
Would this be a good candidate for `go:generate`? https://pkg.go.dev/cmd/go#hdr-Generate_Go_files_by_processing_source

Reply:
Yes, good call. https://github.com/square/certigo/pull/279